beacon_node/client/src/lib.rs

```diff
@@ -143,6 +144,18 @@ where
             None
         };
 
+        // Start the `rest_api` service
+        let _api_exit_signal = if client_config.rest_api.enabled {
```

Comment:
We should probably add an `api_exit_signal: Option<Signal>` field to `Client` so that it's like the others, even though the exit signals never seem to be used.

We should probably also propagate the error, but using error chain looks like a pain so you could just do `.ok()` on the result to get an option.